opentracing-core/src/java/io/opentracing/OpenTracer.java

```diff
@@ -1,19 +0,0 @@
-package io.opentracing;
```

Comment:
these classes miss git compare on format, Only change besides that and adding license header was a missing import in one of them.